packages/react-core/src/components/TextInputGroup/TextInputGroup.tsx

```diff
@@ -11,6 +11,8 @@ export interface TextInputGroupProps extends React.HTMLProps<HTMLDivElement> {
   isDisabled?: boolean;
   /** Flag to indicate the toggle has no border or background */
   isPlain?: boolean;
+  /** Status variant of the text input group. */
+  status?: 'success' | 'warning' | 'error';
```

Comment:
Should we have a 'default' value users can pass similar to `<TextInput>`? https://github.com/patternfly/patternfly-react/blob/2b462a97d110da79a2dc865df34cb1800663a0d8/packages/react-core/src/components/TextInput/TextInput.tsx#L55

Reply:
We could, though we would need to decide if we really want to include "default" as an option for a status prop; FormControlIcon and MenuToggle only have these 3 (MenuToggle has danger instead of error, which is another thing we need to align on).

Reply:
I noticed we have "default" as a value for `<TextInput>` https://github.com/patternfly/patternfly-react/blob/30e3c0351127628ce69c479a7f3157277fb9e7d1/packages/react-core/src/components/TextInput/TextInput.tsx#L55

FWIW I asked about a case for that value at some point, and I think it was to provide an easy value to pass to basically unset the prop (like when the input no longer needs a status because it's good or whatever) instead of having to either conditionally rendering the prop or passing `undefined` or something.

Reply:
It does introduce a potential namespace conflict if there is a "default" variation of something. Our "custom" status used to be called "default" and one of the reasons we changed it was because of the naming conflict it created.

Reply:
Might also bring up the question of whether the prop in TIG should be `validated` to match, or if TextInput should be `status`.

In either case, a value of "none" might be an alternative to "default", if we wanted a means to 'unset' the prop that wasn't `undefined`/conditionally rendering it.

Reply:
I think `status` is what we use for stuff like this more often.

One thought, should the example be called `with status` instead of `with validation` if we stick with the status name?

Reply:
FYI created https://github.com/patternfly/patternfly-react/issues/10851 as a followup to cover a bulk of this. Did update the example name here, though.

Reply:
hmmm.... should we keep the name same as text input for now and update it with #10851?
Or we could deprecate "validated" prop and add the status prop.